Formula/cromshell-alpha.rb

```diff
@@ -0,0 +1,55 @@
+class CromshellAlpha < Formula
```

Comment:
Homebrew has a "versions" syntax to allow multiple formulas of the same tool to exist.  We should maybe use that?    https://docs.brew.sh/Versions

I'm not sure if it's a good idea or not.  I think it helps with the problem of how do we transition from this to a full release without it breaking people's brew installation.  I don't know if it allows for the new version to have a new name.

Reply:
Sounds like the syntax is the name of the tool followed by `@` then the version. So for this instance, it would be
`cromshell@2.alpha.rb` and `class CromshellAT2Alpha < Formula` for the first line in the file.